return_dynamic_filters.php

```diff
@@ -135,19 +93,25 @@ function return_dynamic_filters_prepend_headers() {
 $t_function_name = 'print_filter_' . utf8_substr( $f_filter_target, 0, -7 ); # -7 for '_filter'
 if( function_exists( $t_function_name ) ) {
 	return_dynamic_filters_prepend_headers();
-	call_user_func( $t_function_name );
+	call_user_func( $t_function_name, $t_filter );
```

Comment:
right...

change the function call to this one, which is defined in filter_form_api.php:
`function filter_form_get_input( array $p_filter, $p_filter_target, $p_show_inputs = true )`
that function was made to skip the special cases like thar

like this, i think:
`call_user_func( 'filter_form_get_input', array( $t_filter, filter_field_name  ) );`

are you working on this, so i wait for you to have it fixed?